File: src/main/java/com/jss/jssbackend/modules/tiers/service/BillingItemService.java
package com.jss.jssbackend.modules.tiers.service;

import java.util.List;

import com.jss.jssbackend.modules.tiers.model.BillingItem;

public interface BillingItemService {
    public List<BillingItem> getBillingItems();

    public BillingItem getBillingItem(Integer id);
}

File: client/src/app/modules/tiers/services/billing.item.service.ts
import { HttpClient, HttpParams } from '@angular/common/http';
import { Injectable } from '@angular/core';
import { AppRestService } from 'src/app/appRest.service';
import { BillingItem } from '../../tiers/model/BillingItem';

@Injectable({
  providedIn: 'root'
})
export class BillingItemService extends AppRestService<BillingItem>{

  constructor(http: HttpClient) {
    super(http, "tiers");
  }

  getBillingItems() {
    return this.getList(new HttpParams(), "billing-items");
  }

}
